src/resources/Restoration/BC.ts

```diff
@@ -0,0 +1,52 @@
+import { RestorationResourceIF } from '@/interfaces'
+import { CorpTypeCd, FilingCodes, RuleIds } from '@/enums'
+import { RestorationSteps } from './steps'
+import { GetCorpFullDescription } from '@bcrs-shared-components/corp-type-module'
+
```

Comment:
A lot of this will need to be updated as the UI designs are finished and as each page is worked on.

Ditto for the resource files for the other 3 entity types.